### PR TITLE
Update acknowledgements

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

- Add acknowledgement
- Remove comment

### Related issues

#481 

### Suggested reviewers (optional)

@yemarshall

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -511,6 +511,8 @@ authors:
     name: Yael Evelyn Marshall
     initials: YEM
     orcid: 0000-0003-3723-9683
+    affiliations:
+      - Independent Researcher
```
Comment (yemarshall) [inline content/metadata.yaml:515]: @rando2 I think you didn't see my comment for what to write for affiliations. For some reason I don't see an option to edit this file, so please put under affiliations these 2 things (& remove "independent researcher"):
  - freelance copyeditor
  - recent PhD, Princeton University

Comment (yemarshall) [reply content/metadata.yaml:515]: @rando2 I put in the affiliations myself through PR #546 so please remove this file from this PR but keep the acknowledgments file above it & use the other PR to update the affiliations correctly. thanks :-) I was trying to make things less messy not more messy but as long as it gets done :-)

**Comment by rando2** [comment]
@yemarshall it's extremely hard to keep track of comments on closed PRs, please just open new PRs in the future.

**Comment by rando2** [comment]
@yemarshall can you review and mark "approve" if you approve this in its current form? Thank you!

File: content/70.coi-contribs.md [hunk 2]
```diff
@@ -15,5 +15,4 @@
 ### Acknowledgements
 
 We thank Nick DeVito for assistance with the Evidence-Based Medicine Data Lab COVID-19 TrialsTracker data.
-We are grateful to the following contributors for reviewing pieces of the text: James Eberwine and Ipsita Krishnan.
-<!-- Confirm Nadia Danilova acknowledgement for help with #481 -->
+We are grateful to the following contributors for reviewing pieces of the text: Nadia Danilova, James Eberwine and Ipsita Krishnan.
```
Comment (yemarshall) [inline content/70.coi-contribs.md:18]: @rando2 Acknowledgements update looks good to me. thanks.

**Pull request merged**